### PR TITLE
Select working directory from template

### PR DESCRIPTION
**Pull request by alexKazarin**
Compiled jar validated locally, works as expected.
Let me know if you can see some side-effects (especially about escaping |).

### CODE REVIEW

**Comment by alexKazarin** [comment]
@TaylorOno could you please have a look

File: src/main/java/com/github/idea/ginkgo/util/GinkgoUtil.java [hunk 1]
```diff
@@ -56,7 +56,10 @@ private static Optional<String> getSpecDescription(GoCallExpr ginkgoSpecFunction
     }
 
     public static String escapeRegexCharacters(String specName) {
-        return specName.replace("\"", "").replace("(", "\\(").replace(")", "\\)");
+        return specName.replace("\"", "")
+                .replace("(", "\\(")
+                .replace(")", "\\)")
+                .replace("|", "\\\\|");
```
Comment (TaylorOno) [inline src/main/java/com/github/idea/ginkgo/util/GinkgoUtil.java:62]: Can you update the tests to capture this [here](https://github.com/IdeaGinkgo/Intellij-Ginkgo/blob/master/src/test/java/com/github/idea/ginkgo/util/GinkgoUtilTest.java#L34)

Comment (alexKazarin) [reply src/main/java/com/github/idea/ginkgo/util/GinkgoUtil.java:62]: Updated, test pass successfully
image

**Comment by alexKazarin** [comment]
@TaylorOno could you please check

**Review by TaylorOno — APPROVED** [review]

**Pull request merged**